### PR TITLE
Add helm values yaml metadata to some addons

### PR DESCRIPTION
**Pull request by shaneutt**
This PR adds some links to the annotations of several addons which indicate that the addon has default helm chart values and links to a view of those defaults.

Resolves [DCOS-59121](https://jira.mesosphere.com/browse/DCOS-59121)

### CODE REVIEW

**Comment by s12chung** [comment]
think you want the raw file don't you?

**Comment by joejulian** [comment]
Not only might you want the raw contents, but you should probably also get from the same version as the chart (basically an impossible task through git/github).

Probably the best way to get that would be by either processing the index.yaml, ie:
```
$ yq '.entries["awsebscsiprovisioner"][] | select(.version == "0.1.0").urls[0]' index.yaml
https://mesosphere.github.io/charts/stable/awsebscsiprovisioner-0.1.0.tgz
```
then pulling the values.yaml from the tarball, or using `helm fetch ` and pulling values.yaml from the tarball.

File: templates/traefik-forward-auth.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ spec:
     minSupportedVersion: v1.15.0
   requires:
     matchLabels:
-      kubeaddons.mesosphere.io/name: dex    
+      kubeaddons.mesosphere.io/name: dex
```
Comment (hectorj2f) [inline templates/traefik-forward-auth.yaml]: why didn't you add it here ?

**Comment by shaneutt** [comment]
Rebasing and WIP for a bit

File: templates/traefik-forward-auth.yaml (hunk 1 above)
Comment (shaneutt) [reply templates/traefik-forward-auth.yaml]: will add this in the rework.

**Comment by shaneutt** [comment]
Ready for re-review, added RAW urls and found explicit refs which match the versions.

**Review by joejulian — APPROVED** [review]

**Comment by shaneutt** [comment]
Follow-up for future improvements (we need this now due to time constraints): [DCOS-59231](https://jira.mesosphere.com/browse/DCOS-59231)

**Comment by alejandroEsc** [comment]
I dont agree with this as the logic for this can be part of kommander itself. I will not block though.

**Pull request merged**